ynput/OpenPype
# Max: User's setting for scene unit scale

## Changelog Description
Options for users to set the default scene unit scale for their scenes.
AYON
![image](https://github.com/ynput/OpenPype/assets/64118225/74e017ff-6c1b-4b2b-a19d-baf0d21af964)
![image](https://github.com/ynput/OpenPype/assets/64118225/9e9bbee0-703d-4b01-8b64-987ebdfbbac2)

Legacy OP
![image](https://github.com/ynput/OpenPype/assets/64118225/e3539ec5-5903-4db1-8510-a78f0d4cdbe6)
![image](https://github.com/ynput/OpenPype/assets/64118225/4b72f2e1-ae5d-4343-b62a-d31ac25ea488)

## Additional info
Fbx extractor is refactored regarding to the implementation for the related unit scale parameters.


## Testing notes:
Legacy OP
1. Enable Set Unit Scale
2. Choose what to set as default unit scale for scene
3. Launch Max, it should be set accordingly to what you set in OP settings
AYON
1. Build the addons with this branch
2. Install the Max and Openpype addons which is built from this branch
3. Repeat the steps of "Legacy OP" above.
